### ISSUE TITLE
Please add support for CHD files on RG350

### ISSUE THREAD

**Issue by hehk1234** [gameblabla/pcsx4all#21]
I test your emulator, I can tell you is very optimized and fast.... only bloody roar 2 I need set frame skip to 1 value to get 60 fps... but all games is running at full speed. 
I miss CHD support, as it can run with a quick read and also compresses games better than CSO. I would love for you to take a look at this and if possible add this support. Thanks for your work here bro.

**Comment by gameblabla** [comment]
Well too bad that only the libretro core supports CHD files. I wouldn't keep my hopes up for merging back CHD support just saying.

**Comment by hehk1234** [comment]
OK Alright. Your last PSX emulator, I can say it drastically improved 007 world is not enough, but there is a problem with it, I can't save it, it keeps saying "saved failed, out of disk space?"

**Comment by hehk1234** [comment]
https://gameblabla.nl/files/ipk/gcw0/

**Comment by gameblabla** [comment]
Yeah, i noticed the issue with save states. Unfortunately, the same change that improves its speed also breaks save states. So for now, i reverted back to it. It should now both have the 30 FPS issue fix and have working saves. Download the same OPK again

I'm hoping for senquack to release his build because it gets tiring...

**Comment by hehk1234** [comment]
ok, it's saving now, but the analogs are not working correctly ....

**Comment by gameblabla** [comment]
I'm very confused as i literally did no changes... Are you sure its not disabled in the settings ?

**Comment by hehk1234** [comment]
I found the problem, if I change anything related to digital, dualanalog, dualshock, it hangs all joystick settings. That in the case I am forced to put to digital to work the analog again, but I lose vibration system, because I am not using the dualshock function. Can you test there?
I can get the analog to work again by deleting the PSX configuration folder.
